src/app/src/main/java/com/cmput301f21t09/budgetprojectname/HabitModel.java

```diff
@@ -0,0 +1,39 @@
+package com.cmput301f21t09.budgetprojectname;
+import java.util.Date;
+
+//TODO: Add Frequency
+public class HabitModel {
+    final private String title;
+    final private String ID;
+    final private String reason;
+    final private int streak;
+    final private Date startDate;
+    private Date lastCompleted;
```

Comment:
Why is this not final?

Reply:
if lastCompleted can be reassigned, then it shouldn't be final right?

Reply:
actually w/e I'll just change it give me a sec